backend/api/v1/v1_mobile/views.py

```diff
@@ -374,22 +374,19 @@ def get_queryset(self):
             required=True,
             type=OpenApiTypes.NUMBER,
             location=OpenApiParameter.QUERY),
-        OpenApiParameter(
-            name='administration',
-            required=False,
-            type=OpenApiTypes.NUMBER,
-            location=OpenApiParameter.QUERY),
     ],
     tags=['Mobile Device Form'],
     summary='GET Download List for Syncing Datapoints',
 )
 @api_view(['GET'])
 @permission_classes([IsMobileAssignment])
 def get_datapoint_download_list(request, version):
+    assignment = cast(MobileAssignmentToken, request.auth).assignment
+    administrations = [a.id for a in assignment.administrations.all()]
```

Comment:
`assignment.administrations.values('id')`. please also check what is the different so you understand @ifirmawan

Reply:
`assignment.administrations.values('id')` is more efficient because it only grabs the IDs directly from the database, without loading the whole objects into memory. So, for fetching just a list of IDs, this approach will save on memory and speed up the process.

Reply:
Noted, based on the documentation here
https://docs.djangoproject.com/en/5.0/ref/models/querysets/#values
Yes, this function returns dictionaries, rather than model instances